app/views/shared/_landing_strip.html.haml

```diff
@@ -1,6 +1,6 @@
 .landing-strip
   = t('landing_strip_html',
     name: content_tag(:span, display_name(account), class: :emojify),
-    domain: site_hostname,
-    link_to_root_path: link_to(content_tag(:strong, site_hostname), root_path),
-    sign_up_path: new_user_registration_path)
+    link_to_root_path: link_to(content_tag(:strong, site_hostname), root_path))
+  - if open_registrations?
```

Comment:
hmm...
To make this pr, at first, I tried to use `InstancePresenter` to get the configuration (and DRY).
but IMHO, it's not a very good idea to provide the presenter instance only for `_landing_strip.html.haml` .

Now, `_landing_strip.html.haml` is called from

- `app/views/accounts/show.html.haml` ( `app/controllers/accounts_controller.rb` )
- `app/views/stream_entries/show.html.haml` (`app/controllers/stream_entries_controller.rb` )

There two controllers are completely difference (unlike `about_controller`), so I think it's hard for developers to know how controller's `InstanceRepresenter` instance variable is used.

Of course, I'll add both controllers by your request, but my opinion is above.